<launch>
  <include file="$(find dogsim)/launch/base.launch" />
  <node pkg="dogsim" type="robot_driver" name="robot_driver" output="screen"/>
  <node pkg="dogsim" type="high_arm_position_action" name="adjust_dog_position_action" output="screen"/>
</launch>
